build.gradle

```diff
@@ -1,4 +1,5 @@
 import org.apache.tools.ant.filters.ReplaceTokens
+
```

Comment:
As long as the line breaks stay the same, I can configure the diff viewer to ignore those changes.